### ISSUE TITLE
I have my ssh key set up. Still get dependencies error 

### ISSUE THREAD

**Issue by daoran** [ivalab/meta_ClosedLoopBench#1]
```shell
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
image_geometry: Cannot locate rosdep definition for [opencv3]
delayed_odometry: Cannot locate rosdep definition for [kobuki_gazebo_plugins]
turtlebot_trajectory_testing: Cannot locate rosdep definition for [common_rosdeps]
kobuki_capabilities: Cannot locate rosdep definition for [rocon_apps]
turtlebot_description: Cannot locate rosdep definition for [create_description]
cv_bridge: Cannot locate rosdep definition for [opencv3]
turtlebot_bringup: Cannot locate rosdep definition for [rocon_app_manager]
```
I have passed my ssh connection test, and I use ROS melodic.
How can I solve this problem?

**Comment by YipuZhao** [comment]
We never tested it under ROS Melodic, but it seems to be related.  OpenCV has been removed from ROS distribution after Kinetic.  Other missing rosdep might due to the same reason.

The easiest way to get it working would be down-grading to ROS Kinetic.  Otherwise you might need to find corresponding packages for Melodic.

**Comment by daoran** [comment]
Hello, thanks for your help. 
I have used docker to try again. 
I have to fixed some issues to get it work.

1. Move `turtlebot_trajectory_testing` package outside workspace(then move back), otherwise `rosdep install --from-paths src -i -y` cannot install corresponding ROS packages. 

2. I have to resolve ros-kinetic-librealsense issue. This is related to the linux kernel version. Since I used Ubuntu 16.04.5, I have to install librealsense2-dkms. More info can be found in this [url](https://github.com/IntelRealSense/librealsense/issues/2402)

3. CMakeLists.txt of some packages inside vision_opencv requires opencv 3.4.1, and set path to `/opt/opencv3`

4. I removed some packages, `cv_teset`, `gf_orb_slam2`

It seems working. I will let you know if I find more issues.
